android-maps-utils-ktx/src/main/java/com/google/maps/ktx/kml/Kml.kt

```diff
@@ -0,0 +1,53 @@
+package com.google.maps.ktx.kml
+
+import androidx.annotation.RawRes
+import androidx.fragment.app.FragmentActivity
+import com.google.android.gms.maps.GoogleMap
+import com.google.maps.android.collections.GroundOverlayManager
+import com.google.maps.android.collections.MarkerManager
+import com.google.maps.android.collections.PolygonManager
+import com.google.maps.android.collections.PolylineManager
+import com.google.maps.android.data.kml.KmlLayer
+import java.io.InputStream
+
+/**
+ * Alias for the [KmlLayer] constructor that provides Kotlin named parameters and default values.
+ */
+inline fun kmlLayer(
+    map: GoogleMap,
+    @RawRes resourceId: Int,
+    activity: FragmentActivity,
```

Comment:
I'd suggest merging https://github.com/googlemaps/android-maps-utils/pull/631 and then update this to take in a `Context` and adding the cache parameter.

Reply:
Yep, agreed

Reply:
Updated fa1f839